tests/ui/span/issue-35987.stderr

```diff
@@ -8,11 +8,6 @@ LL | impl<T: Clone, Add> Add for Foo<T> {
    |                ---  ^^^ not a trait
```

Comment:
An aside: I'm realizing now, looking at the code in question (and confirmed by the first line of the description of issue #35987):

```rust
impl<T: Clone, Add> Add for Foo<T> { ... }
```

that the most likely explanation for what has happened is that someone was trying to put both the `Clone` and `Add` bounds onto the type parameter `T`, and did not realize that the traits needed to be separated by `+` rather than `,`.

In an ideal world, with full LLM linguistic modelling of the user's *intent*, we might hope to provide a hint that the comma, while syntatically correct, could instead have been a `+`.

(I know that Vadim [already pointed out](https://github.com/rust-lang/rust/issues/35987#issuecomment-242387111) that the compiler "can't" complain about the comma in all cases, since it is syntactically correct there. I'm just saying to potentially give to the developer.)